docs/interpreter.md

```diff
@@ -9,23 +9,25 @@
 stores the `mlir::ShapedType` of the tensor along with a contiguous byte array
 representing its data laid out in
 [major-to-minor order](https://www.tensorflow.org/xla/shapes). `detail::Buffer`
-objects are reference-counted to simplify memory management.
+objects are reference-counted to simplify memory management. A `Tensor` object
+also stores [stride](https://en.wikipedia.org/wiki/Stride_of_an_array) for each
+dimension to interpret the underlying 1-D storage as a multi-dimensional tensor.
```

Comment:
Do we have to: 1) store strides in the `Tensor` instead of computing them on the fly, 2) call them out in the document? Strides are an internal implementation detail of major-to-minor order which doesn't even need to be computed explicitly (its only use is `flattenIndices`, and there you can just have a running counter instead of an entire vector).

Reply:
removed stride as member variable. updated the document accordingly.